README.md

```diff
@@ -16,8 +16,9 @@ Total Coverage: <b>99.39%</b>
 ## Inputs
 
 ##### `github-token` (**Required**)
```

Comment:
Can you remove the `(**Required**)` tag on this header too?

Reply:
Done